code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -94,6 +94,13 @@
 	icon_state = "nanite_hypo"
 	list_reagents = list(/datum/reagent/mutationtoxin/supersoldier = 5)
 
+/obj/item/reagent_containers/hypospray/combat/supersoldier_syndie
+	name = "Syndicate Supersoldier Nanites"
+	desc = "A reverse engineered nanotrasen supersoldier injector."
+	item_state = "nanite_hypo_syndie"
+	icon_state = "nanite_hypo_syndie"
+	list_reagents = list(/datum/reagent/mutationtoxin/supersoldier/syndie = 5)

```

Comment:
Could I not take the reagents out of the injector, and make multiple pills of this, making it useable by multiple people?

Reply:
i changed the amount of toxin to 1